README.md

```diff
@@ -9,7 +9,44 @@ running in a cluster, such as rolling upgrades.
 To very concisely describe Aurora, it is like a distributed monit or distributed supervisord that
 you can instruct to do things like _run 100 of these, somewhere, forever_.
 
-
+## Medallia Features
+
+* Jobs using Docker containerizer doesn't use thermos-executor:
+  - Removes the requirement of having Python in the Docker Image.
+  - Use Docker image default CMD if the process cmdline is empty.
+* Different Instances of a Job can have different Docker Parameters. 
+  - specify a list of `Instance` in the `instance_variables` in the Job config.
+  - each `Instance` has a `variables` attribute as a List of ```Variable``` which are name-value pairs
+    that are used to parameterize instances. For example, the following configuration will spawn three 
+    instances of the myService job assigning different values to the ip-address and volume Docker 
+    parameters for each instance. The `instances` attribute is optional and is infered from the 
+    number of `instance_variables`. 
+     
+```python
+       Service(cluster       = 'myCluster',
+          role               = 'myRole',
+          environment        = 'devel',
+          name               = 'myService',
+          instances          = 3,
+          task               =  hello_world_task,
+          instance_variables = [
+               Instance(variables=[
+                  Variable(name="ip", value="10.1.1.1"),
+                  Variable(name="vol", value="vol-A")]),
+               Instance(variables=[
+                  Variable(name="ip", value="10.1.1.2"),
+                  Variable(name="vol", value="vol-B")]),
+               Instance(variables=[
+                  Variable(name="ip", value="10.1.1.3"),
+                  Variable(name="vol", value="vol-C")])
+               ],
```

Comment:
Just talked in person about this. We could add an example about how to do it with a function in the .aurora file directly.